dhis-2/build-dev.sh

```diff
@@ -38,8 +38,8 @@ print() {
 
 print "Building dhis2-core..."
 
-mvn clean install -T1C -DskipTests=true -f $DIR/pom.xml
-mvn clean install -T1C -DskipTests=true -f $DIR/dhis-web/pom.xml
+mvn clean install -T1C -Pdev -Pjdk11 -f $DIR/pom.xml
```

Comment:
does it still skip tests ?

Reply:
Yes, skipTest on/off is now set explicitly on all profiles, dev has =false

Reply:
we need true then